app/components/LoginPage.js

```diff
@@ -29,6 +29,9 @@ export default class LoginPage extends Component {
     this.setState({ email: '',
                     password: ''
                   })
+
+
+    console.log(this.state.history)
```

Comment:
`this.state` is referring to the state of this component (`LoginPage`). The only thing I see on this state is `email` and `password` (in the constructor).